src/zenml/model_registries/base_model_registry.py

```diff
@@ -62,14 +63,24 @@ class ModelRegistryModelMetadata(BaseModel):
     model and its development process.
     """
 
-    zenml_version: Optional[str] = None
+    _managed_by: str = "zenml"
```

Comment:
Neither this field nor its associated property are actually used anywhere. Is there something missing ?

Reply:
It will return this as a set of key-value when `ModelRegistryModelMetadata` object is called